### PR TITLE
Support multiple patient referrals

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: db/health/schema.rb [hunk 1]
```diff
@@ -861,6 +863,7 @@
     t.integer "rejected_reason", default: 0, null: false
     t.integer "patient_id"
     t.integer "accountable_care_organization_id"
+    t.datetime "effective_date", default: -> { "now()" }
```
Comment (eanders) [inline db/health/schema.rb:866]: This looks suspicious.

Comment (awisema) [reply db/health/schema.rb]: Agreed! I merged pre-release and ran `bin/update` to make sure the schemas were up to date...

File: spec/factories/health/patients.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@
     birthdate { Date.current }
     sequence(:medicaid_id)
     enrollment_start_date { Date.current }
+    current { true }
```
Comment (eanders) [inline spec/factories/health/patients.rb:8]: I might have missed this earlier, but isn't tithe referral that should have current?  There should be many referrals per patient I think.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@
   ],
 )
 # SimpleCov.start 'rails'
-SimpleCov.start
+# SimpleCov.start
```
Comment (eanders) [inline spec/rails_helper.rb:17]: This looks like a mistake.

**Comment by awisema** [comment]
Combined w/ #853

**Pull request closed without merging**